documentation/modules/security/proc-accessing-kafka-using-loadbalancers.adoc

```diff
@@ -65,5 +65,5 @@ kubectl get kafka my-cluster -o=jsonpath='{.status.listeners[?(@.name=="external
 [source,shell,subs=+quotes]
 kubectl get secret _KAFKA-CLUSTER-NAME_-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
 +
-Use the extracted certificate in your Kafka client to configure TLS connection.
-If you enabled any authentication, you will also need to configure SASL or TLS authentication.
+Use the extracted certificate in your Kafka client to configure the TLS connection.
+If you enabled any authentication, you will also need to configure SASL or mTLS authentication.
```

Comment:
Again, maybe we can make this more generic to not list any specific authentication types?